src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -7863,17 +7863,14 @@ namespace NS2
                 state.Workspace.GlobalOptions.SetGlobalOption(CompletionOptionsStorage.ForceExpandedCompletionIndexCreation, True)
                 state.Workspace.GlobalOptions.SetGlobalOption(CompletionOptionsStorage.ShowItemsFromUnimportedNamespaces, LanguageNames.CSharp, True)
 
-                ' trigger completion with import completion disabled
                 Await state.SendInvokeCompletionListAndWaitForUiRenderAsync()
 
                 ' make sure expander is selected
-                Await state.SetCompletionItemExpanderStateAndWaitForUiRenderAsync(isSelected:=True)
```

Comment:
My hypothesis: The expander button should already be selected here, therefore setting it would cause no UI change. Waiting for UI render then ended up with cancellation because of timeout.